README.md

```diff
@@ -1079,6 +1079,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Siddharth Sabale](https://github.com/siddharth-sable)
   - [Siddharth Singh Chouhan](https://github.com/Sid672)
   - [Siddharth Singh Tanwar](https://github.com/SiddharthSinghTanwar)
+  - [Sidra Waseem](https://github.com/sidramwaseem)
```

Comment:
Please add your name in line 1084, after Sideon Kaves.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable